lib/Ama/Model/Votes.pm

```diff
@@ -30,4 +37,18 @@ sub uncast {
   $@ ? { error => $@ } : $results;
 }
 
+sub _autoRemove {
```

Comment:
The method, however, still needs to get called.  So even tho it's auto to the user, it's only auto thru the cast action.  The remove method itself doesn't do anything "auto", it simply removes the records you tell it to via parameters.  Keep in mind that the whole point of pulling that out into its own function is code reuse, that means not every action that may wish to remove a vote will do so in an "auto" way.